tests/test_data.py

```diff
@@ -0,0 +1,88 @@
+from pathlib import Path
+
+from tdc.multi_pred import DTI
+from torchvision import datasets
+
+from kale.loaddata.mnistm import MNISTM
+from kale.loaddata.usps import USPS
+from kale.prepdata.image_transform import get_transform
+from kale.utils.download import download_file_by_url
+
+
+def download_path():
+    path = Path("tests") / "test_data"
+    path.mkdir(parents=True, exist_ok=True)
+    return str(path)
+
+
+path_test_data = download_path()
+
+
+# tests/conftest.py
```

Comment:
The comments are not clear. Please use a consistent format throughout, e.g. 'Downloading gait gallery data for tests/conftest.py test'. Please change all the comments below.